exonum-java-binding/core/rust/integration_tests/src/vm.rs

```diff
@@ -31,41 +31,37 @@ const CONVERSION_FAILED_MESSAGE: &str = "Failed to convert FS path into utf-8";
 
 /// Creates a configured `JavaVM` for benchmarks.
 /// _`JavaVM` should be created only *once*._
-#[allow(dead_code)]
 pub fn create_vm_for_benchmarks() -> Arc<JavaVM> {
     Arc::new(create_vm(false, false))
 }
 
-/// Creates a configured `JavaVM` for benchmarks with fake classes.
+/// Creates a configured `JavaVM` for benchmarks with EJB classes.
 /// _`JavaVM` should be created only *once*._
-#[allow(dead_code)]
-pub fn create_vm_for_benchmarks_with_fakes() -> Arc<JavaVM> {
+pub fn create_vm_for_benchmarks_with_classes() -> Arc<JavaVM> {
     Arc::new(create_vm(false, true))
 }
 
-/// Creates a configured `JavaVM` for tests.
+/// Creates a configured `JavaVM` for tests with EJB classes.
 /// _`JavaVM` should be created only *once*._
-#[allow(dead_code)]
-pub fn create_vm_for_tests() -> Arc<JavaVM> {
-    Arc::new(create_vm(true, false))
-}
-
-/// Creates a configured `JavaVM` for tests with fake classes.
-/// _`JavaVM` should be created only *once*._
-#[allow(dead_code)]
-pub fn create_vm_for_tests_with_fake_classes() -> Arc<JavaVM> {
+pub fn create_vm_for_tests_with_classes() -> Arc<JavaVM> {
     Arc::new(create_vm(true, true))
 }
 
 /// Creates a configured `JavaVM`.
 /// _JavaVM should be created only *once*._
-fn create_vm(debug: bool, with_fakes: bool) -> JavaVM {
+///
+/// If `debug` is true, additional checks of correctness of JNI operations are
+/// enabled.
+///
+/// If `with_classes` is true, Java classes, dependencies of the EJB App are
```

Comment:
I think it's fine now, but ideally these ITs must use core classpath so that they can be launched right after building core, and not after app. They do not depend on Rust app.

Reply:
ECR-4094 created